server/src/main/java/io/crate/execution/ddl/tables/AddColumnTask.java

```diff
@@ -74,6 +75,7 @@ public AddColumnTask(NodeContext nodeContext, CheckedFunction<IndexMetadata, Map
     @Override
     @SuppressWarnings("unchecked")
     public ClusterState execute(ClusterState currentState, AddColumnRequest request) throws Exception {
+        addedColumns.clear();
```

Comment:
Why do we need to clear it? Afaik on each request a new task is created https://github.com/crate/crate/pull/14635/files#diff-adf9a95543f9a5054a6c767e1d3ba548312481197879fc41924f06b69bbaf664R80